feat : 로그인 페이지 디자인 제작

sion0221 opened the pull request:
## PR 유형

- [X] 새로운 기능 추가
- [ ] 버그 수정
- [ ] CSS 등 사용자 UI 디자인 변경
- [x] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [X] 코드 리팩토링
- [ ] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [X] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

## PR 체크리스트

<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->

- [X] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [X] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### PR 상세
로그인 페이지 디자인 제작
- 클래스네임 명확하게 구분지어서 수정 및 파일과 폴더 위치 수정
- button 공통 컴포넌트에 타입 추가(button, submit, reset), 버튼 속성으로 classname에 undefined 명시 안할 시, 오류가 발생함으로서 수정함

## 이슈

<!-- 이슈 키워드와 함께 #을 입력한 후 이슈 번호를 선택해주세요. -->
<!-- 에시 : resolves #1 -->

resolves #15

sion0221 committed:
feat : 로그인 페이지 디자인 제작

sion0221 committed:
Merge remote-tracking branch 'origin/develop' into feature/login/heo

sion0221 committed:
refactor : 폴더 및 파일 위치 수정 및 클래스네임 수정
- app 폴더에 있던 컴포넌트 폴더 및 파일 위치를 외부로 컴포넌트 폴더에 수정
- 클래스 네임 명확하게 다시 수정

sion0221 committed:
refacor : aria-label 값 정정
- 표시가 아닌 내용 삭제이므로, 올바르게 다시 수정

sion0221 committed:
refactor : img 태그 -> image 컴포넌트로 수정

chajiiiii commented on bun.lock:
```diff
@@ -4,6 +4,7 @@
     "": {
       "name": "next-js-manual-template",
       "dependencies": {
+        "next-js-manual-template": ".",
```
이거 뭐에요??

HeoSion replied:
잘 모르겠네요.. 일단 불필요한거 지워서 다시 올릴게요!

chajiiiii commented on src/app/auth/login/page.module.css:
```diff
@@ -0,0 +1,63 @@
+/* 로그인 페이지 전체 컨테이너 */
```
파일명을 page.module.css로 하는건 어떨까요?

chajiiiii commented on src/app/auth/login/page.module.css:
```diff
@@ -0,0 +1,63 @@
+.loginPageContainer {
+  width: 100%;
+  min-width: 393px;
+  max-width: 768px;
```
피그마 시안에는 폼 최대 너비 481px까지 입니다~~

HeoSion replied:
넵 수정하겠습니다~!

chajiiiii commented on src/components/atom/input/input.tsx:
```diff
@@ -0,0 +1,93 @@
+      />
+
+      {clearable && isEmail && value && (
+        <button
```
label 안에 버튼들을 밖으로 빼면 좋을 거 같습니다.
https://developer.mozilla.org/ko/docs/Web/HTML/Reference/Elements/label#%EC%A0%91%EA%B7%BC%EC%84%B1_%EA%B3%A0%EB%A0%A4%EC%82%AC%ED%95%AD

HeoSion replied:
구조 다시 수정해서 올릴게요!

chajiiiii reviewed (commented):
코멘트 남겼으니 확인 부탁드립니다 :)

sion0221 committed:
refactor : 오타 코드와 크기 수정 및 삭제, bun.lock 코드 수정
- 오타가 있었던 코드를 삭제
- max-width 크기 768px -> 481px 로 수정, min-width 크기도 시안에 맞게 363px로 수정
- bun.lock 내용 불필요한 코드 수정

sion0221 committed:
refactor : bun.lock, pakage.json 불필요한 의존성 삭제

sion0221 committed:
refactor : 누락된 파일주소 수정

sion0221 committed:
style : 최소너비 삭제 및 최대 너비 수정
- 최소너비 길이 스타일 제거, 최대 너비 481px로 수정
